src/locales/fr/PAGES/AUDIT.json

```
@@ -0,0 +1,20 @@
+{
+    "ASSERTION_RESULT_DESCRIPTION_LABEL": "Observations",
+    "BTN_EXPAND_PAGES": "Afficher les pages Web pour saisir des résultats individuels",
+    "HD_SAMPLE_SELECT": "Ajouter des résultats aux pages",
+    "HOW_TO": "Comment satisfaire à",
+    "INTRO": "Enregistrez le résultat de l’évaluation des pages Web sélectionnées à l’étape précédente. Comparez les résultats entre la page structurée et les pages sélectionnées au hasard, et ajustez si nécessaire l’échantillon sélectionné à l’étape précédente.",
+    "IMPORT_DATA_INTRO": "Ajouter des données provenant d’autres tests (automatisés)",
+    "RESOURCE_LINK_NAME": "Étape 4 : Auditer l’échantillon sélectionné",
+    "LABEL_OUTCOME": "Résultat",
+    "NO_SAMPLE": "Aucun échantillon disponible",
+    "RESULTS_FOR": "Résultats pour ",
+    "SAMPLE_FINDINGS": "Échantillon complet",
+    "SELECT_ALL": "Sélectionner tout",
+    "TITLE": "Étape 4 : Auditer l’échantillon complet",
+    "UNDERSTAND": "Explications",
```

```suggestion
    "UNDERSTAND": "Comprendre",
```
@KodGit I forgot to mention this one. 

Context: is used to link to a specific resource explaining each success criterion. Example: "Understanding 1.1.1".

In WCAG 2.1. French Authorized Translation, "Comprendre" is used. See for example https://www.w3.org/Translations/WCAG21-fr/#audio-only-and-video-only-prerecorded